src/AddItems.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import { db } from './lib/firebase';
+
+function AddItems(props) {
+  const [groceryItem, setGroceryItem] = useState('');
+  const userToken = localStorage.getItem('userToken');
+  const [itemFreq, setItemFreq] = useState(7);
+
+  const updateGroceryItem = (event) => {
+    setGroceryItem(event.target.value);
+  };
+
+  const submitGroceryItem = (event) => {
+    event.preventDefault();
+    const formData = {
+      itemName: groceryItem,
+      frequency: itemFreq,
+      lastPurchaseDate: null,
+    };
+    db.collection(userToken).add({ formData });
+    setGroceryItem('');
+    setItemFreq(7);
+  };
+
+  const radioBtnHandler = (event) => {
+    setItemFreq(+event.target.value);
+  };
+
+  return (
+    <div>
+      <form id="test-form" onSubmit={submitGroceryItem}>
+        <label htmlFor="item">Item name:</label>
+        <input
+          type="text"
+          id="item"
+          value={groceryItem}
+          required
+          onChange={updateGroceryItem}
+        />
+        <fieldset>
```

Comment:
```suggestion
        <fieldset>
          <legend>How soon would you like to buy it?</legend>
```
Fieldset should have a `legend` associated with it...then it's golden 🔥